src/Referent/ManagedUsersFilter.php

```diff
@@ -59,6 +60,7 @@ class ManagedUsersFilter
      * @Assert\NotNull
      */
     protected $offset = 0;
+    protected $managedArea = '';
```

Comment:
what about prefixing by `query` as other query parameters? 
maybe just `queryZone`? (same remark for Entity parameter)